### ISSUE TITLE
aria-keyshortcuts should use AltGraph instead of Option

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#4325]
## Version

`main` of `4.15.3-0`

## Describe the bug

[According to W3C](https://w3c.github.io/aria/#aria-keyshortcuts), `aria-keyshortcuts` should be `"alt+shift+a"` or `"ctrl+altgraph+a"` (for "Ctrl + Option + A" in macOS)

However, when narrating it as accessible name, we do not need + (plus) signs.

| Keystroke | `aria-keyshortcuts` | Accessible name |
| - | - | - |
| <kbd>Alt</kbd> + <kbd>Shift</kbd> + <kbd>A</kbd> | `"Alt+Shift+a"` | `"Alt Shift A"` |
| <kbd>Ctrl</kbd> + <kbd>Option</kbd> + <kbd>A</kbd> | `"Ctrl+AltGraph+a"` | `"Ctrl Option A"` |

## Steps to reproduce

1. Send `suggested-actions` to the bot
2. Observe the `<button>` element of suggested action button

## Expected behavior

It should have `aria-keyshortcuts` of `"alt+shift+a"`.

## Additional context

[Bug]
